src/rccl_wrap.cc

```diff
@@ -52,6 +52,9 @@ void rcclUpdateCollectiveProtocol(struct ncclComm* comm, size_t const& nBytes, s
       // When LL128 is performant, the next condition overrides the previous LL choice
       if (comm->topo->ll128Enabled) {
         if (info->func == ncclFuncAllReduce) {
+          if(comm->nNodes > 2) {
+            ll128Max *= 15.2; // Scale max message size for n > 2 since Tree has special behavior at 2 nodes
```

Comment:
What are the LL128 thresholds for 2 nodes vs. > 2 nodes?

Reply:
Shared